### PR TITLE
Handle error and also correct error strings.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: cmd/nfs-subdir-external-provisioner/provisioner.go [hunk 1]
```diff
@@ -125,7 +129,7 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 			AccessModes:                   options.PVC.Spec.AccessModes,
 			MountOptions:                  options.StorageClass.MountOptions,
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)],
+				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[(v1.ResourceStorage)],
```
Comment (kmova) [inline cmd/nfs-subdir-external-provisioner/provisioner.go:132]: @humblec -- q_ why is this change required?

Comment (humblec) [reply cmd/nfs-subdir-external-provisioner/provisioner.go]: keeping it intact for consistency... ptal. thanks.

File: cmd/nfs-subdir-external-provisioner/provisioner.go [hunk 2]
```diff
@@ -155,14 +159,12 @@ func (p *nfsProvisioner) Delete(ctx context.Context, volume *v1.PersistentVolume
 	}
 
 	// Determine if the "onDelete" parameter exists.
-	// If it exists and has a delete value, delete the directory.
-	// If it exists and has a retain value, safe the directory.
+	// If it exists and has a deleted value, delete the directory.
+	// If it exists and has a retail value, safe the directory.
```
Comment (kmova) [inline cmd/nfs-subdir-external-provisioner/provisioner.go:163]: n_: retain

Comment (humblec) [reply cmd/nfs-subdir-external-provisioner/provisioner.go]: oh, corrected and made those values to be explicit in the comments. @kmova ptal. thanks

**Review by kmova — APPROVED** [review]
/lgtm
/approve
